<!-- # TODO: #6 Fix table caption font--><!-- # TODO: #7 Fix citations links font size--><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Design | iGEM Stockholm</title><link href="../css/content.css" rel="stylesheet"></head><body><!-- # TODO: #6 Fix table caption font--><!-- # TODO: #7 Fix citations links font size--><nav class="navbar navbar-expand-xl fixed-top"><div class="container d-flex justify-content-between"><a class="navbar-brand d-lg-inline-block" href="//"></a><button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"><i class="fas fa-bars"></i></button><div class="collapse navbar-collapse" id="navbarNav"><ul class="navbar-nav ml-auto"><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarProjectDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Project</a><div class="dropdown-menu fade-up" aria-labelledby="navbarProjectDropdown"><a class="dropdown-item" href="//Description">Description</a><a class="dropdown-item" href="//Design">Design</a><a class="dropdown-item" href="//Proof_Of_Concept">Proof of Concept</a><a class="dropdown-item" href="//Implementation">Implementation</a><a class="dropdown-item" href="//Contribution">Contribution</a><a class="dropdown-item" href="//Poster">Poster</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarLab &amp; PartsDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Lab &amp; Parts</a><div class="dropdown-menu fade-up" aria-labelledby="navbarLab &amp; PartsDropdown"><a class="dropdown-item" href="//Model">Model</a><a class="dropdown-item" href="//Engineering">Engineering</a><a class="dropdown-item" href="//Parts">Parts</a><a class="dropdown-item" href="//Experiments">Experiments</a><a class="dropdown-item" href="//Results">Results</a><a class="dropdown-item" href="//Safety">Safety</a><a class="dropdown-item" href="//Notebook">Notebook</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarHuman PracticesDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Human Practices</a><div class="dropdown-menu fade-up" aria-labelledby="navbarHuman PracticesDropdown"><a class="dropdown-item" href="//Human_Practices">Human Practices</a><a class="dropdown-item" href="//Education">Education</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarAboutDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">About</a><div class="dropdown-menu fade-up" aria-labelledby="navbarAboutDropdown"><a class="dropdown-item" href="//Team">Team</a><a class="dropdown-item" href="//Collaborations">Collaborations</a><a class="dropdown-item" href="//Sponsors">Sponsors</a><a class="dropdown-item" href="//Attributions">Attributions</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarAwardsDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Awards</a><div class="dropdown-menu fade-up" aria-labelledby="navbarAwardsDropdown"><a class="dropdown-item" href="//Entrepreneurship">Entrepreneurship</a><a class="dropdown-item" href="//Sustainable">Sustainable</a><a class="dropdown-item" href="//Medal_Criteria">Medal Criteria</a></div></li></ul></div><div class="d-flex" id="themeSwitchWrapper"><i class="far fa-sun"></i><div id="themeSwitch"><label class="switch" for="themeSwitchInput"><input type="checkbox" id="themeSwitchInput"><span class="slider round"></span></label></div><i class="far fa-moon"></i></div></div></nav><div class="parent"><header class="d-flex justify-content-center align-items-center"><div class="container"><h1>Design</h1><p class="lead pl-1">S-POP in a nutshell</p><a class="arrow" href="#wave"><i class="fas fa-chevron-down"></i></a><img class="topimg" src="../assets/img/Contents--topimg_design.png"><div class="wave wave1" id="wave"></div><div class="wave wave2"></div><div class="wave wave3"></div></div></header><main><div class="container"><div class="row"><div class="sidebar col-lg-3"><div class="nav" id="contents"><h5>Contents</h5><ul></ul></div></div><div class="content col-lg-9"><article><h1>Introduction</h1><p>The engineering design is the primary step in the engineering cycle. The general workflow goes as follows: defining a problem, conducting background research, specifying requirements, brainstorming and developing a prototype solution. After these steps comes testing the solution, communicating and evaluating the results to improve the prototype based on the results. This process is similar to the usual scientific method where the focus is on answering a question but the engineering process instead develops a solution to a specific problem <a href="#citation1">Comparing the Engineering Design Process and the Scientific Method, n.d.</a>. In the design process for S-POP, we wanted to focus on being able to test our parts, modularity, and predictable behavior. Ensuring that all parts could be tested as smaller components was a necessity as our system is complex. In glossary literature research concerns were raised of the predictability and stability of biological systems. Therefore predictability became one of the points of interest for our project.</p><p>The S-POP-system was divided into two main components, the modular biosensor and <em>S. oneidensis</em> oscillation module, both which later on would be contained in a microbial fuel cell (MFC). By creating this modular biosensor, it would be possible to adapt the monitoring system to detect other compounds while maintaining the oscillation.</p><h1>Biosensor</h1><p>The modular biosensor has gone through several design iterations, but the main idea of utilizing a biological sensing part (bioreceptor) in a modular system remained intact throughout the entire project. The modularity of the biosensor opens up the possibility of different uses for the system. With the modularity, other bioreceptors can be introduced and utilized to detect other compounds as well. By utilizing synthetic biology the biosensor could be adapted and optimized for other purposes. Several different quorum sensing molecules (QS) were considered and evaluated. The most important criteria regarding the choice of QS molecules were mobility through cell membranes and low crosstalk with each other, as well as potential already existing systems in our strains. The QS molecules are of great importance for the <em>S. oneidensis</em> oscillation-system, therefore the choice was more dependent on this system rather than the biosensor. Since the biosensor has fewer parts, it is more adaptable to the requirements imposed by the <em>S. oneidensis</em> oscillation-system.</p><p>When considering the different sensing parts, we mainly looked at older iGEM projects to build on the foundation that others have paved. We decided to focus on persistent organic pollutants which are a current problem for the Baltic sea. This decision was based on literature and conversations with advisors and experts in the environmental field. We retained PCBs and PFOS as targets for our project. Regulations for the levels of PFOS have been developed in later years for Swedish water treatment plants. But the primary focus in wastewater treatment regulations is on heavy metals and nutrients not POPs <a href="#citation2">Petterson et al., 2020</a>. Since these pollutants are less prioritized more can pass into the environment. The current measurement method most commonly used for POPs is testing water samples in the lab.</p><p>In order to develop S-POP into a fully functioning system all components needed to be tested. To make these tasks more manageable the system was divided up into:</p><ol><li>Detection</li><li><em>S. oneidensis</em> oscillation-system</li><li>MFC</li></ol><p>For the detection of PFOS and PCBs the genetic constructs were separated into several smaller parts in order to check individual promoters and gene expression. Appropriate controls were used to check the function of each construct. Constructs used throughout the experiments are listed below.</p><h1>Detection</h1><h1>Detection of PFOS</h1><h2>The prmA promoter system and its characterization</h2><p>The bio-receptor for PFOS detection, prmA, is an inducible promoter from <em>Rhodococcus jostii</em> RHA1 that should be induced in the presence of 100Î¼M PFOS. This genetic part is already established in the iGEM system and was designed by Nathan Young, from the 2019 USAFA iGEM team <a href="BBa_K2911000">BBa_K2911000</a>. LuxI, the autoinducer synthase for AHL, to be inserted after the prmA in order to produce the chosen QS molecule, 3OC6HSL which would bind to LuxR in <em>S. oneidensis</em>. system <a href="BBa_C0061">BBa_C0061</a>. PrmA being a previously established part the functionality can be tested by adding fluorescence (GFP) after prmA induction. In the final system for PFOS detection, the QS molecule AHL produced in the PFOS-detection system which should bind to the QS recipient (LuxR) which induces the <em>S. oneidensis</em> oscillation system as illustrated in Figure 1.</p><div class="image"><img src="../assets/img/Design--PFOS_circuit.png" alt="PFOS detection by prma promoter" style="width: 60%"><p>Figure 1: PFOS detection by prma promoter</p></div><ol><li><p>(Pconst-RBS-GFP-DT)<a href="http://parts.igem.org/Part:BBa_K3440011">BBa_K3440011</a> In order to evaluate if the desired genes had been inserted into <em>E. coli</em> and were expressed correctly, several different genetic constructs were used. A constitutive promoter was utilized. The constitutive promoter was checked by adding a region of fluorescent molecule (GFP) that will be expressed under a functional constitutive promoter.</p></li><li><p>(Pconst-RBS-LuxI-Myc-DT)<a href="http://parts.igem.org/Part:BBa_K3440000">BBa_K3440000</a><br>To show proper LuxI expression, it was put under a constitutive promoter and a Myc-tag added to LuxI to visualize expression with Western Blot.</p></li><li><p>(Pprma-RBS-mCherry-DT)<a href="http://parts.igem.org/Part:BBa_K3440002">BBa_K3440002</a> Firstly the bio-receptor, prmA, PFOS inducible promoter's leakiness had to be evaluated. This was performed by tagging it with fluorescence (mCherry). Some leakiness is expected for promoters and by checking prmA by itself it is then possible to compare the results to the full construct.</p></li></ol><h2>Full construct</h2><ol start="4"><li>(Pprma-RBS-LuxI-Myc)<a href="http://parts.igem.org/Part:BBa_K3440005">BBa_K3440005</a><br>The final construct for the PFOS detection module combines the prmA promoter and LuxI with a Myc-tag in the end to evaluate expression by Western blot. This construct was utilized to check that LuxI could be expressed under the prmA promoter while the expression in the presence/absence of PFOS was also tested. Figure 2 illustrates the full genetic construct utilized in PFOS detection. The activation of prmA by the presence of PFOS induces the expression of the QS molecule AHL (3OC6-HSL).</li></ol><div class="image"><img src="../assets/img/Design--prma.png" alt="Full PFOS construct" style="width: 100%"><p>Figure 2: Full PFOS construct</p></div><p>The QS molecule of interest would be transferred to the <em>S. oneidensis</em> oscillation-system and bound to LuxR. For the detection of PCBs, a similar system is utilized with the exception of the promoter region which can be adapted according to the application. However, the production and detection of the QS molecules is the common factor in the identification of PFOS and PCBs.</p><h1>Detection of PCBs</h1><h2>The bphR1/2 system and its characterization</h2><p>The initial plan was to use a group of PCBs to test our experimental design. However, for safety reasons and according to the current regulation, it had to be replaced by a similar but less hazardous compound. Hence, we decided to replace PCBs by 1,1-biphenyl as described in <a href="https://2020.igem.org/Team:Stockholm/Safety">Safety</a>. The bio-receptor used in the detection of 1,1-biphenyl is a combination of bphR1 promoter <a href="http://parts.igem.org/Part:BBa_K1155001">BBa_K1155001</a>, a positive regulator for bphR1 protein and bphR2 <a href="http://parts.igem.org/Part:BBa_K1413021">BBa_K1413021</a>. In the absence of 1,1-biphenyl, small amounts of bphR2 protein binds to bphR2 operator to repress bphR2 transcription (autorepression) and activate bphR1 weakly. When 1,1-biphenyl is present in the media, bphR2 protein binds to bphR1 operators to strongly activate their transcription. When the transcription is activated, the QS molecule is produced and binds to LuxR in the <em>S. oneidensis</em>. system as illustrated in Figure 3.</p><div class="image"><img src="../assets/img/Engineering--PCB circuit.png" alt="PCB detection by bphR1 promoter" style="width: 100%"><p>Figure 3: PCB detection by bphR1 promoter</p></div><ol start="5"><li><p>(Pconst-RBS-bphR2-Myc-DT)<a href="http://parts.igem.org/Part:BBa_K3440009">BBa_K3440009</a><br>In order to evaluate if the genes of interest had been inserted into <em>E. coli</em> and were expressed correctly, several different genetic constructs were utilized. Firstly bphR2 expression was checked by putting under a constitutive promoter and a Myc-tag at the end of the construct to evaluate expression by performing Western Blot.</p></li><li><p>(PbphR1-RBS-mCherry-DT)<a href="http://parts.igem.org/Part:BBa_K3440001">BBa_K3440001</a> In order to evaluate the leakiness of bphR1 promoter as well as checking the expression in the presence/absence of 1,1-biphenyl, gene expressing fluorescent molecule (mCherry) was tagged onto the construct.</p></li><li><p>(Pbphr1-RBS-LuxI-Myc)<a href="http://parts.igem.org/Part:BBa_K3440010">BBa_K3440010</a><br>To check the expression of LuxI under the bphR1 promoter a Myc-Tag was tagged to the construct. The construct was tested in the presence/absence of PCBs and expression was evaluated with Western blot.</p></li><li><p>(Pconst-RBS-bphR2-Myc-DT-Pbphr1-RBS-mCherry)<a href="http://parts.igem.org/Part:BBa_K3440017">BBa_K3440017</a> The PCB detection relies on the activation of the bphR1 promoter by the expression of bphR2. bphR1 expression should be stronger in the presence of PCB. In order to evaluate the expression Myc-Tag and fluorescence (mCherry) was added to the construct.</p></li></ol><h2>Full construct</h2><ol start="9"><li>(Pconst-RBS-bphR2-Myc-DT-Pbphr1-RBS-LuxI)<a href="http://parts.igem.org/Part:BBa_K3440018">BBa_K3440018</a> The full construct for PCB detection and expression of LuxI was to be inserted in <em>E. coli</em> and expression tested with <em>Chromobacterium Violaceum</em>, that turns purple in the presence of AHLs. This system is illustrated in Figure 4, with a constitutive promoter that constantly induces bphR2, in the presence of PCBs, the bphR2 forms a complex with PCB. This complex upregulates the inducible promoter bphR1 which induces the expression of the QS molecule AHL (3OC6-HSL).</li></ol><div class="image"><img src="../assets/img/Design--bphr1.png" alt="PCB detection, full construct" style="width: 80%"><p>Figure 4: PCB detection, full construct</p></div><p>The sensing parts were separated from the oscillation system to simplify experimental design. The oscillation system was then also separated into several smaller parts in order to check the individual promoters, QS-molecule binding and QS-molecule production. The electrical output capabilities of <em>S. oneidensis</em> were tested separately within a MFC.</p><h1><em>Shewanella oneidensis</em> oscillation module for electrical output</h1><p>The general concept of the oscillation circuit integrates the key part (RhlR) under an inducible promoter and has the reporter gene under the pRhl promoter. The reporter gene in the final circuit would be MtrB, which in the knockout strain of <em>S. oneidensis</em> would complete the Mtr complex and allow it to produce electricity.</p><p>The inducible promoter would be the pLux promoter, which together with LuxR can receive 3OC6-HSL from <em>E. coli</em>. When there is no 3OC6-HSL present in the sample, there should be no induction of pLux, which in turn would result in no transcription of RhlR. The oscillatory circuit is then incomplete and should not produce any signal. The RhlR concentration here is not static. To properly utilize <em>S. oneidensis</em> Oscillation-system illustrated in Figure 5, it was necessary to characterize all the parts involved in the genetic oscillator.</p><div class="image"><img src="../assets/img/Model--Overview_New.png" alt="Oscillation system in *S. oneidensis*" style="width: 100%"><p>Figure 5: Oscillation system in *S. oneidensis*</p></div><p>To test the entire oscillation-system the individual genetic parts and their expression needed to be checked. For simplicity a majority of the constructs were to be tested in <em>E.coli</em>, with the exception of the final full construct, and a construct that utilised MtrB as a reporter gene.</p><h2>LuxI/R - characterization</h2><ol start="10"><li><p>(Plux-RBS-GFP)<a href="http://parts.igem.org/Part:BBa_K3440003">BBa_K3440003</a> The expression of the promoter Plux is upregulated by the LuxR activator protein complexed with the autoinducer HSL. In order to evaluate the leakiness of this promoter fluorescence (GFP) was tagged on to check expression.</p></li><li><p>(Pconst-RBS-LuxR-Myc)<a href="http://parts.igem.org/Part:BBa_K3440013">BBa_K3440013</a> To show the expression of LuxR it was put under a constitutive promoter and tagged with a Myc-Tag and evaluated by Western Blot. In complex with HSL, LuxR binds to the Plux promoter, activating transcription.</p></li><li><p>(Pconst-RBS-LuxR-Myc-DT-Plux-RBS-GFP)<a href="http://parts.igem.org/Part:BBa_K3440014">BBa_K3440014</a> In order to show that the LuxR can be expressed and activate the Plux promoter in presence of AHL. LuxR was put under a constitutive promoter and Plux tagged with fluorescence (GFP) to evaluate expression of the entire construct.</p></li></ol><p>The full oscillation system requires the autoinducers LuxI and RhlI and their respective receptors LuxR and the RhlR to be fully functional, as well as their specific promoters Plux and Prhl. The entire system contains a lot of different genetic parts that needed to be characterized. Due to time constraints not all constructs were characterized.</p><h2>Rhl/R - characterization</h2><ol start="13"><li><p>(Prhl-RBS-GFP)<a href="http://parts.igem.org/Part:BBa_K3440004">BBa_K3440004</a> The promoter Prhl is activated by the RhlR activator protein in concert with the auto-inducer C4-HSL which is produced by RhlI. In order to evaluate the leakiness of this promoter fluorescence (GFP) was tagged on to check expression.</p></li><li><p>(Pconst-RBS-RhlI-Myc)<a href="http://parts.igem.org/Part:BBa_K3440006">BBa_K3440006</a> To check the expression of RhlI it was put under a constitutive promoter and tagged with a Myc-tag to evaluate the expression with Western Blot.</p></li><li><p>(Pconst-RBS-RhlR-Myc-DT)<a href="http://parts.igem.org/Part:BBa_K3440008">BBa_K3440008</a> By putting RhlR, activator protein, under a constitutive promoter and tagged with a Myc-tag the expression can be evaluated with Western Blot.</p></li><li><p>(Pconst-RBS-RHlR-Myc-DT-Prhl-RBS-GFP)<a href="http://parts.igem.org/Part:BBa_K3440012">BBa_K3440012</a><br>To show that RhlR expression is dependent on the activation of Prhl promoter by AHL, Prhl tagged with fluorescent molecule (GFP) was used to evaluate leakage of the promoter.</p></li></ol><p>The full construct of the oscillation module can only produce an electrical output with <em>S. oneidensis</em> in a MFC.</p><h2>Full construct</h2><p>The oscillatory circuit consists of a receptor (RhlR) which can receive its cognate ligand (C4-HSL) to activate an inducible promoter (pRhl). An AHL synthase (RhlI) producing the cognate ligand is under the control of the inducible promoter, creating a positive feedback loop. Under the same promoter, a lactonase (AiiA) is also controlled, which degrades the cognate ligand, creating a negative feedback loop.</p><p>The presence of C4-HSL together with RhlR activates pRhl, which produces more RhlI synthase, which in turn creates more C4-HSL. The positive feedback loop amplifies the C4-HSL signal until enough AiiA has been synthesised. As the lactonase starts degrading the C4-HSL, the signal decreases. The negative feedback loop decreases the C4-HSL signal until no new protein is produced and the remaining active protein is broken down.</p><ol start="17"><li><p>(Pconst-RBS-MtrB-Myc)<a href="http://parts.igem.org/Part:BBa_K3440016">BBa_K3440016</a> MtrB is an outer membrane protein, found in the extracellular iron oxide respiratory system in <em>S. oneidensis</em>, required for the metal reducing process. In order to perform a successful MtrB expression. It was put under a constitutive promoter and tagged with a Myc-tag. The expression of MtrB was evaluated by Western Blot.</p></li><li><p>(Pconst-RBS-AiiA-Myc)<a href="http://parts.igem.org/Part:BBa_K3440007">BBa_K3440007</a> AiiA is the coding region for autoinducer inactivation enzyme A which catalyzes the degradation of N-acyl-homoserine lactones (AHLs), quorum sensing autoinducers. In order to check the expression of AiiA, it was put under a constitutive promoter and was tagged with a Myc-tag. Hence, its expression was evaluated by Western Blot.</p></li><li><p>(Pconst-RBS-LuxR-DT-Plux-RBS-MtrB-DT)<a href="http://parts.igem.org/Part:BBa_K3440019">BBa_K3440019</a> This larger construct was designed to show that MtrB could be expressed by Plux in the presence of AHL. In our system, MtrB, expressed by <em>S. oneidensis</em>, acts as the reporter by increasing the production of current in a microbial fuel cell.</p></li><li><p>(Pconst-RBS-RhlR-DT-Prhl-RBS-RhlI-RBS-AiiA-RBS-GFP-DT)<a href="http://parts.igem.org/Part:BBa_K3440020">BBa_K3440020</a><br>The second to last construct in the oscillation system. This construct includes all the genes of interest except from the reporter MtrB gene. MtrB gene was exchanged by GFP gene in order to evaluate the expression of the oscillation-system in <em>E. coli</em> by fluorescence. This genetic modified <em>E.coli</em> strain would act as a representative construct that would not be dependent on <em>S. oneidensis</em>. MtrB as a reporter gene is the only viable option for <em>S.oneidensis</em> in a MFC, since it would produce an electric signal.</p></li><li><p>(Plux-RBS-MtrB-DT-Pconst-RBS-RhlR-DT-Prhl-RBS-RhlI-RBS-AiiA-RBS-LuxR-DT)<a href="http://parts.igem.org/Part:BBa_K3440021">BBa_K3440021</a> The final construct for the <em>S. oneidensis</em> oscillation-system was to be implemented in a MFC with MtrB as the reporter gene producing electricity.</p></li></ol><h1>MFC</h1><p>In many biological systems, the energy used is in the form ATP which is formed by breaking down nutrients. The energy extracted from glycolysis and the Krebs cycle is not primarily in the form of ATP, but NADH, which is then used in the electron transport chain in oxidative phosphorylation. In some anaerobic bacteria such as <em>S. oneidensis</em>, an extracellular electron transport pathway still exists, but without oxygen as a terminal electron acceptor to form water, oxidized metals or other compounds can be reduced instead. This can result in electricity being generated as the reduced mediator can pass its electron on.<a href="#citation3">Pirbadian, S. et al, 2014</a>. A microbial fuel cell (MFC) is a fuel cell system that utilizes bio-electrochemistry to drive an electric current with bacteria<a href="#citation4">Schmidt-Rohr, K., 2020</a>.</p><h2><em>Shewanella oneidensis</em> MR-1</h2><p><em>Shewanella oneidensis</em> MR-1 is known as a &quot;Dissimilatory Metal-Reducing Bacteria (DMRB)&quot; due to its metal reducing ability that is coupled to its metabolism <a href="#citation5">Lloyd, J., 2003</a>. Dissimilatory metal reduction is a process used by microbes that conserves energy through oxidation of organic or inorganic electron donors and reduction of metals and metalloids.</p><p>This process creates an electrochemical gradient which provides the chemical energy for growth. When used in an MFC the natural electrochemical gradient can provide a detectable electric signal. <a href="#citation6">Schicklberger, M., 2013</a>. <em>Shewanella Oneidensis</em> MR-1 can also form highly conductive nanowires that can carry charge far outside the immediate area of the cell and remove the need for specific mediators. <a href="#citation7">Reguera, G. et al, 2005</a>. This is in <em>Shewanella oneidensis</em> done through a set of cytochromes that carry the electrons along the nanowire <a href="#citation3">Pirbadian, S. et al, 2014</a> C-type cytochromes are utilized by <em>S. oneidensis</em> as electron transfer proteins and flavins to facilitate the electron transport process. This is a common strategy used by a variety of DMRB bacteria to transfer electrons to the cell surface. It's a characteristic to have a high number of c-type cytochrome encoding genes for metal reducing bacteria <a href="#citation8">Beblawy, S. et al., 2018</a>. MtrB, after transporting through the cytoplasmic membrane and across the periplasm, becomes a part of a porin-cytochrome complex in the outer membrane in <em>S. oneidensis</em>. In MtrB-gene knockout studies <em>S. oneidensis</em> metal reducing capabilities are significantly diminished <a href="#citation9">Schuetz, B. et al., 2009</a>. By taking advantage of the essential nature of MtrB we can compare the electric output and biofilm formation in <em>S. oneidensis</em> MR-1 between a wild type from and a MtrB knockout strain.</p><p><em>S. oneidensis</em> can thrive in both aerobic and anaerobic conditions (facultative bacteria) and can utilize common carbon sources for energy. To optimize the metal reducing capabilities and biofilm formation, some conditions should be fulfilled. For solely the survival of <em>S. oneidensis</em> most carbon sources are sufficient, e.g. lactate and furumate. Under anaerobic conditions lactate is a preferable carbon source that benefits biofilm formation <a href="#citation10">Pinchuk, G., 2011</a>.</p><h2>Fuel cell design</h2><p>Fuel cells convert chemical energy into electrical energy via a pair of redox reactions. These systems can have many different designs depending on the need. Included but not limited to, proton exchange membrane fuel cells, electro-galvanic fuel cells and zinc air battery. Two available setups for the MFC were a H-cell and a flow cell. An H-cell physically separates anodic and cathodic reactions by means of a porous membrane. An issue with the available H-cell setup was the distance between the two electrodes creating a large internal resistance. H-cells are more commonly used in systems with homogenous systems, meaning both electrode chambers contain either solution or gas. The H-cells are set up as a batch system while the flow cell has a continual flow. The flow cell setup available was deemed the most favorable for our system. A flow cell setup can be utilized as a hybrid system or homogeneous-system. The hybrid setup having one of the electrode sides exposed to gas and the other exposed to liquid. Separated by a porous membrane proton exchange is facilitated. The hybrid flow cell system was used.</p><p>For MFCs, special considerations need to be taken into account such as low flow rates, temperature and usage of solvents that the bacteria can survive in. <em>S. oneidensis</em> biofilm formation is a vital aspect for the electric signal production leading to the prioritization of the conditions required for proper biofilm formation.</p><h2>Mediated/Undemediated</h2><p>Generally there are two categories of MFCs, mediated and unmediated. A mediated system relies on a chemical compound to facilitate electron transport to the anode, while an unmediated system relies on the bacterias active redox proteins and electron transport via the outer membrane directly to the anode. <a href="#citation11">Badwal, S., 2014</a>. In the early stages of this project both a mediated and an unmediated system were considered. For optimization of the MFC it was planned to test both but due to time constraints, only the unmediated system could be tested. As well as testing mediated and unmediated systems, the supplementation of ferric ions in the MFC had been planned since it can significantly increase the electricity generation. Neither was utilized in the end <a href="#citation12">Wu, D. et al., 2013</a>.</p><h2>Size</h2><p>When designing the MFC two different sizings of fuel cells were particularly interesting. Some research has indicated that a higher power density can be produced in a mini-MFC with a chamber volume of ca 1mL. With the smaller distance to transport electrons the internal resistance decreases likely leading to a stronger signal. <a href="#citation13">Ringesen, B. et al., 2007</a>. Though after evaluating the available equipment it was decided to utilize an already established fuel cell. But for future research a point of interest should be a mini-MFC system.</p><h2>Final design of the MFC</h2><div class="image"><img src="../assets/img/Experiments--MFC_Schematic.png" alt="Schematic of the MFC" style="width: 100%"><p>Figure 6: Schematic of the MFC</p></div><p>The hybrid fuel flow cell that was utilized in the setup, illustrated in Figure 6. The cathode side is exposed to air, while the anode side was in liquid solute. Both electrodes were carbon paper that had been thermally treated to be hydrophobic. Sandwiched between the anode and cathode is a proton exchange membrane. The catholyte, inflow of air and the analyte inflow of liquid media (LB) inoculated with wild type <em>S. oneidensis</em>. The bacteria naturally form a biofilm on the electrode. Since it is in a continuous flow cell, we were unable to disassemble it during runs.The biofilm was to be visually inspected after the runs were concluded. When a biofilm forms on the electrode, a detectable increase in voltage output should be visible. In similar MFC setups the external resistance ranged between 200-10kâ¦. Due to limited available equipment an external resistance of 1kâ¦ was deemed most appropriate.</p></article><article id="references"><h1>References</h1><ol><li id="citation1"><p class="author"></p><cite>Comparing the engineering design process and the scientific method. Science Buddies.</cite><p><span class="details">(n.d.). Retrieved on October 27, 2020. from </span><a class="in-text url" href="https://www.sciencebuddies.org/science-fair-projects/engineering-design-process/engineering-design-compare-scientific-method" target="_blank" rel="noopener">https://www.sciencebuddies.org/science-fair-projects/engineering-design-process/engineering-design-compare-scientific-method</a></p><a class="in-text" href="#intext1">Back to text</a></li><li id="citation2"><p class="author"></p><cite>PreliminÃ¤ra riktvÃ¤rden fÃ¶r PFAS (poly- och perfluorerade alkylsubstanser) i mark och grundvatten. Statens geotekniska institut.</cite><p><span class="details">(2020). Retrieved on October 27, 2020. from </span><a class="in-text url" href="https://www.swedgeo.se/sv/om-sgi/vart-samhallsuppdrag/regeringsuppdrag/riktvarden-pfas/" target="_blank" rel="noopener">https://www.swedgeo.se/sv/om-sgi/vart-samhallsuppdrag/regeringsuppdrag/riktvarden-pfas/</a></p><a class="in-text" href="#intext2">Back to text</a></li><li id="citation3"><p class="author">Pirbadian, S., Barchinger, S. E., Leung, K. M., Byun, H. S., Jangir, Y., Bouhenni, R. A., ... &amp; El-Naggar, M. Y. (2014).</p><cite>Shewanella oneidensis MR-1 nanowires are outer membrane and periplasmic extensions of the extracellular electron transport components.</cite><p><span class="journalTitle">Proceedings of the National Academy of Sciences</span> <span class="journalInfo">111(35), 12883-12888.</span></p><a class="in-text" href="https://doi.org/10.1073/pnas.1410551111" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Shewanella oneidensis MR-1 nanowires are outer membrane and periplasmic extensions of the extracellular electron transport components." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext3">Back to text</a></li><li id="citation4"><p class="author">Schmidt-Rohr, K. (2020).</p><cite>Oxygen Is the High-Energy Molecule Powering Complex Multicellular Life: Fundamental Corrections to Traditional Bioenergetics.</cite><p><span class="journalTitle">ACS Omega</span> <span class="journalInfo">5(5), 2221-2233.</span></p><a class="in-text" href="https://doi.org/10.1021/acsomega.9b03352" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Oxygen Is the High-Energy Molecule Powering Complex Multicellular Life: Fundamental Corrections to Traditional Bioenergetics." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext4">Back to text</a></li><li id="citation5"><p class="author">Lloyd, J. R. (2003).</p><cite>Microbial reduction of metals and radionuclides.</cite><p><span class="journalTitle">FEMS Microbiology Reviews</span> <span class="journalInfo">27(2-3), 411-425.</span></p><a class="in-text" href="https://doi.org/10.1016/S0168-6445(03)00044-5" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Microbial reduction of metals and radionuclides." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext5">Back to text</a></li><li id="citation6"><p class="author">Schicklberger, M., Sturm, G., &amp; Gescher, J. (2012).</p><cite>Genomic Plasticity Enables a Secondary Electron Transport Pathway in Shewanella oneidensis.</cite><p><span class="journalTitle">Applied and Environmental Microbiology</span> <span class="journalInfo">79(4), 1150-1159.</span></p><a class="in-text" href="http://dx.doi.org/10.1128/AEM.03556-12" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Genomic Plasticity Enables a Secondary Electron Transport Pathway in Shewanella oneidensis." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext6">Back to text</a></li><li id="citation7"><p class="author">Reguera, G., McCarthy, K. D., Mehta, T., Nicoll, J. S., Tuominen, M. T., &amp; Lovley, D. R. (2005).</p><cite>Extracellular electron transfer via microbial nanowires.</cite><p><span class="journalTitle">Nature</span> <span class="journalInfo">435(7045), 1098-1101.</span></p><a class="in-text" href="https://doi.org/10.1038/nature03661" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Extracellular electron transfer via microbial nanowires." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext7">Back to text</a></li><li id="citation8"><p class="author">Beblawy, S., Bursac, T., Paquete, C., Louro, R., Clarke, T. A., &amp; Gescher, J. (2018).</p><cite>Extracellular reduction of solid electron acceptors by Shewanella oneidensis.</cite><p><span class="journalTitle">Molecular Microbiology</span> <span class="journalInfo">109(5), 571-583.</span></p><a class="in-text" href="https://doi.org/10.1111/mmi.14067" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Extracellular reduction of solid electron acceptors by
            Shewanella oneidensis." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext8">Back to text</a></li><li id="citation9"><p class="author">Schuetz, B., Schicklberger, M., Kuermann, J., Spormann, A. M., &amp; Gescher, J. (2009).</p><cite>Periplasmic Electron Transfer via the c-Type Cytochromes MtrA and FccA of Shewanella oneidensis MR-1.</cite><p><span class="journalTitle">Applied and Environmental Microbiology</span> <span class="journalInfo">75(24), 7789-7796.</span></p><a class="in-text" href="http://dx.doi.org/10.1128/AEM.01834-09" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Periplasmic Electron Transfer via the c-Type Cytochromes MtrA and FccA of Shewanella oneidensis MR-1." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext9">Back to text</a></li><li id="citation10"><p class="author">Pinchuk, G. E., Geydebrekht, O. V., Hill, E. A., Reed, J. L., Konopka, A. E., Beliaev, A. S., &amp; Fredrickson, J. K. (2011).</p><cite>Pyruvate and Lactate Metabolism by Shewanella oneidensis MR-1 under Fermentation, Oxygen Limitation, and Fumarate Respiration Conditions.</cite><p><span class="journalTitle">Applied and Environmental Microbiology</span> <span class="journalInfo">77(23), 8234-8240.</span></p><a class="in-text" href="https://doi.org/10.1128/AEM.05382-11" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Pyruvate and Lactate Metabolism by Shewanella oneidensis MR-1 under Fermentation, Oxygen Limitation, and Fumarate Respiration Conditions." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext10">Back to text</a></li><li id="citation11"><p class="author">Badwal, S. P. S., Giddey, S. S., Munnings, C., Bhatt, A. I., &amp; Hollenkamp, A. F. (2014).</p><cite>Emerging electrochemical energy conversion and storage technologies.</cite><p><span class="journalTitle">Frontiers in Chemistry</span> <span class="journalInfo">2.</span></p><a class="in-text" href="https://doi.org/10.3389%2Ffchem.2014.00079" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Emerging electrochemical energy conversion and storage technologies." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext11">Back to text</a></li><li id="citation12"><p class="author">Wu, D., Xing, D., Lu, L., Wei, M., Liu, B., &amp; Ren, N. (2013).</p><cite>Ferric iron enhances electricity generation by Shewanella oneidensis MR-1 in MFCs.</cite><p><span class="journalTitle">Bioresource Technology</span> <span class="journalInfo">135, 630-634.</span></p><a class="in-text" href="https://doi.org/10.1016/j.biortech.2012.09.106" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Ferric iron enhances electricity generation by Shewanella oneidensis MR-1 in MFCs." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext12">Back to text</a></li><li id="citation13"><p class="author">Ringeisen, B. R., Ray, R., &amp; Little, B. (2007).</p><cite>A miniature microbial fuel cell operating with an aerobic anode chamber.</cite><p><span class="journalTitle">Journal of Power Sources</span> <span class="journalInfo">165(2), 591-597.</span></p><a class="in-text" href="https://doi.org/10.1016/j.jpowsour.2006.10.026" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=A miniature microbial fuel cell operating with an aerobic anode chamber." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext13">Back to text</a></li></ol></article></div></div></div></main><footer><div class="container"><div class="social_media"><p>Follow iGEM Stockholm on social media</p><a href="https://www.instagram.com/igemstockholm/"><i class="fab fa-instagram"></i></a><a href="https://www.facebook.com/igemstockholm"><i class="fab fa-facebook-f"></i></a><a href="https://www.youtube.com/channel/UCh_a6JvWdh6N_i5tYFFcpyw"><i class="fab fa-youtube"></i></a><a href="https://twitter.com/igemstockholm"><i class="fab fa-twitter"></i></a><a href="https://www.linkedin.com/company/igemstockholm"><i class="fab fa-linkedin-in"></i></a><a href="https://igem.se/"><i class="fas fa-globe-europe"></i></a><a href="mailto:info@igem.se"><i class="far fa-envelope"></i></a></div><hr><div class="sponsors"><img class="kth" src="../assets/img/Sponsors--kth.png"><img class="ki_innovations" src="../assets/img/Sponsors--ki_innovations.png"><img class="mf" src="../assets/img/Sponsors--mf.png"><img class="snapgene" src="../assets/img/Sponsors--snapgene.png"><img class="neb" src="../assets/img/Sponsors--neb.png"><img class="idt" src="../assets/img/Sponsors--idt.png"><img class="redbull" src="../assets/img/Sponsors--redbull.png"><img class="benchling" src="../assets/img/Sponsors--benchling.png"></div><hr><div class="credits"><p>Sample template built using the iGEM Wiki Starter Pack by BITS Goa.</p><p>Code released under the MIT license.</p><p>Based on <a href="https://getbootstrap.com">Bootstrap</a> and themes <a href="https://bootswatch.com/flatly/">Flatly</a> and <a href="https://bootswatch.com/darkly/">Darkly</a> from <a href="https://bootswatch.com/">Bootswatch</a>.</p><p>Some content from the <a href="https://2020.igem.org/Team:Example">iGEM Example Wiki</a>. Images from <a href="https://unsplash.com">Unsplash</a>. Web fonts from <a href="https://fonts.google.com">Google</a>.</p></div></div></footer></div><script src="../content.bundle.js"></script></body></html>