<!-- # TODO: #6 Fix table caption font--><!-- # TODO: #7 Fix citations links font size--><html lang="en"><head><meta charset="utf-8"/><meta content="width=device-width,initial-scale=1" name="viewport"/><title>Experiments | iGEM Stockholm</title><script src="https://2020.igem.org/common/MathJax-2.5-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><link href="https://2020.igem.org/Template:Stockholm/css/contentCSS?action=raw&amp;ctype=text/css" rel="stylesheet"/></head><body><!-- # TODO: #6 Fix table caption font--><!-- # TODO: #7 Fix citations links font size--><nav class="navbar navbar-expand-xl fixed-top"><div class="container d-flex justify-content-between"><a class="navbar-brand d-lg-inline-block" href="https://2020.igem.org/Team:Stockholm"></a><button aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation" class="navbar-toggler" data-target="#navbarNav" data-toggle="collapse" type="button"><i class="fas fa-bars"></i></button><div class="collapse navbar-collapse" id="navbarNav"><ul class="navbar-nav ml-auto"><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarProjectDropdown" role="button">Project</a><div aria-labelledby="navbarProjectDropdown" class="dropdown-menu fade-up"><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Description">Description</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Design">Design</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Proof_Of_Concept">Proof of Concept</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Implementation">Implementation</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Contribution">Contribution</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Poster">Poster</a></div></li><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarLab &amp; PartsDropdown" role="button">Lab &amp; Parts</a><div aria-labelledby="navbarLab &amp; PartsDropdown" class="dropdown-menu fade-up"><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Model">Model</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Engineering">Engineering</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Parts">Parts</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Experiments">Experiments</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Results">Results</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Safety">Safety</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Notebook">Notebook</a></div></li><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarHuman PracticesDropdown" role="button">Human Practices</a><div aria-labelledby="navbarHuman PracticesDropdown" class="dropdown-menu fade-up"><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Human_Practices">Human Practices</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Education">Education</a></div></li><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarAboutDropdown" role="button">About</a><div aria-labelledby="navbarAboutDropdown" class="dropdown-menu fade-up"><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Team">Team</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Collaborations">Collaborations</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Sponsors">Sponsors</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Attributions">Attributions</a></div></li><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarAwardsDropdown" role="button">Awards</a><div aria-labelledby="navbarAwardsDropdown" class="dropdown-menu fade-up"><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Entrepreneurship">Entrepreneurship</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Sustainable">Sustainable</a><a class="dropdown-item" href="https://2020.igem.org/Team:Stockholm/Medal_Criteria">Medal Criteria</a></div></li></ul></div><div class="d-flex" id="themeSwitchWrapper"><i class="far fa-sun"></i><div id="themeSwitch"><label class="switch" for="themeSwitchInput"><input id="themeSwitchInput" type="checkbox"/><span class="slider round"></span></label></div><i class="far fa-moon"></i></div></div></nav><div class="parent"><header class="d-flex justify-content-center align-items-center"><div class="container"><h1>Experiments</h1><p class="lead pl-1">Our entire summer</p><a class="arrow" href="#wave"><i class="fas fa-chevron-down"></i></a><img class="topimg" src="https://2020.igem.org/wiki/images/1/15/T--Stockholm--img--Contents--topimg_experiments.png"/><div class="wave wave1" id="wave"></div><div class="wave wave2"></div><div class="wave wave3"></div></div></header><main><div class="container"><div class="row"><div class="sidebar col-lg-3"><div class="nav" id="contents"><h5>Contents</h5><ul></ul></div></div><div class="content col-lg-9"><article><h1>Cloning</h1><p>Cloning can be used to get a specific DNA fragment into the <dfn>vector DNA ~ Small circular chromosome with DNA for functional regions such as origin of replication and a drug resistance gene</dfn> to form a recombinant DNA. This can be introduced into a host cell where the recombinant DNA will replicate and express the proteins. Two enzymes are used in this process <dfn>restriction enzymes ~ Enzymes that cuts DNA at specific sites by recognising 4-8 basepair sequences</dfn> and <dfn>DNA ligase ~ Enzyme that forms phosphodiester bonds between two complementary sticky end DNA fragments</dfn> <a href="#citation1">Lodish et al., 2000</a>.</p><p>The constructs were designed based on prior biobricks or based on literature. Due to the unavailability or late delivery of the biobricks we ordered our constructs from Integrated DNA Technologies (IDT). The constructs under a constitutive promoter had an additional <dfn>Myc tag ~ SsrA tag that tags the protein for degradation</dfn> in the sequence and the promoter specific sequences have either a <dfn>GFP ~ Green Fluorescent Protein often used as a reporter gene</dfn> or <dfn>mCherry ~ Red fluorescent protein based on GFP</dfn> sequence for easier characterization. The full list of constructs can be found in <a href="https://2020.igem.org/Team:Stockholm/Parts">Parts</a>.</p><p>To express these constructs, they were cloned into the standard vector plasmid pSB1C3. A <a href="https://2020.igem.org/wiki/images/b/ba/T--Stockholm--Restriction_digest.pdf">restriction Digest</a> was performed on EcoRI and PstI sites in both the gene blocks and the plasmids. This was followed by <a href="https://2020.igem.org/wiki/images/8/8f/T--Stockholm--Ligation.pdf">ligation</a>. These plasmids were transformed into <em>E coli TOP10</em> and were selected on chloramphenicol plates for <a href="https://2020.igem.org/wiki/images/3/33/T--Stockholm--Colony_PCR.pdf">colony PCR</a>.</p><p>Transformation was primarily performed using <a href="https://2020.igem.org/wiki/images/d/da/T--Stockholm--Transformation.pdf">heat shock</a> of chemically competent <em>E. coli TOP10</em>. However, when heat shock transformation was not possible, electroporation of electrocompetent E coli was done instead. <em>Shewanella Oneidensis</em> can be electroporated, but we opted for conjugation, where <em>E. coli MFDpir</em> was used to transfer plasmids to <em>Shewanella Oneidensis</em>.</p><p>The samples from the colony PCR with the right size when run on an agarose gel were sent for sequencing to Microsynth which were re-checked with the original sequence which if matched were used for characterization.</p><p>Once all the constructs were inserted it was necessary to test if these transformed <em>E. coli TOP10</em> would be able to grow normally in the presence of the pollutants at varying concentrations for which we performed the viability test.</p><div class="image"><img alt="Cloning workflow" src="https://2020.igem.org/wiki/images/3/3e/T--Stockholm--img--Experiments--cloning.png" style="width: 100%"/><p>Figure 1: Cloning workflow</p></div><h1>Viability assays</h1><p>Viability assays were important for our project. In this series of experiments we aimed to investigate any alterations in the growth of our bacteria of interest when cultured with the chemicals of interest.</p><p>When cultured in a closed system, bacteria tend to have a characteristic growth curve (Figure 2). In the lag phase the bacteria adjust to the environment, while in the log phase they grow exponentially with high division rate. However, the log phase is followed by the stationary phase where the division rate is very low and the nutrients are limited. As a result of these facts, the death phase of the culture is followed <a href="#citation2">Wang, Fan, Chen &amp; Terentjev, 2015</a>.</p><div class="image"><img alt="Characteristic bacteria growth curve" src="https://2020.igem.org/wiki/images/8/87/T--Stockholm--img--Experiments--growth_curve.png" style="width: 80%"/><p>Figure 2: Characteristic bacteria growth curve</p></div><p>Growth-based <a href="https://2020.igem.org/wiki/images/5/57/T--Stockholm--Viability_Assay.pdf">viability assays</a> using microplate readers are a quite common technique nowadays. It has partially replaced the traditional methods and has reduced the cost of the experiment. The cultivation of the bacteria is performed in microplates and the growth curve is designed based on the absorbance values. It is a fast, easy and cheap technique to get accurate growth curves. Finally, it allow the user to set different chemical or biological conditions in each well <a href="#citation3">Horakova et al., 2004</a>.</p><h1>Fluorescence</h1><p>Fluorescence emission occurs when a molecule absorbs light at high energy (i.e. short wavelength) and emits light at lower energy and usually visible wavelength <a href="#citation4">Welker, 2012</a>.</p><p>To characterise the activation of specific promoters in our circuit, genes coding for fluorescent proteins (GFP / mCherry) were inserted into some of our constructs. The fluorescence intensity of the constructs were quantified in whole cells, both in uninduced (i.e. to check for "leakiness" of the promoters) and induced conditions. <a href="https://2020.igem.org/wiki/images/6/6e/T--Stockholm--Fluorescence.pdf">These measurements</a> were done in a microplate reader after calibration with fluorescent dyes, and adjusted for cell density.</p><div class="image"><img alt="Basic fluorescense mechanism" src="https://2020.igem.org/wiki/images/e/eb/T--Stockholm--img--Experiments--fluorescence.png" style="width: 80%"/><p>Figure 3: Basic fluorescense mechanism</p></div><h1>SDS Page and Western Blot</h1><p><a href="https://2020.igem.org/wiki/images/b/b3/T--Stockholm--SDS_Page_and_Western_Blot.pdf">Western Blots</a> were utilized for the characterization of our constructs. In order to ensure that our gene insert was in fact expressed and translated into the desired protein we needed a method to detect the proteins that were expressed by our cells. SDS PAGE is an electrophoresis based separation method for size based discrimination of proteins and western blotting utilises antibody specificity in order to detect a specific protein <a href="#citation5">Mahmood &amp; Yang, 2012</a>.</p><p>In order to be able to distinguish our protein of interest from our host cell proteome we designed our gene sequences to include a myc-tag at the end of the protein sequence. The myc-tag is a 1.2 kDa protein and can be fused to the N-terminus or C-terminus of a protein <a href="#citation6">Jarmander et al., 2012</a>.</p><p>With a monoclonal antibody specific for the Myc tag we would be able to determine the presence of our desired protein in different constructs in different conditions, such as with and without the presence of pollutants. A blue safe protein stain could be used in order to stain the general proteome of our samples.</p><h1>Chromobacterium violaceum</h1><p>Quorum sensing is dependent on the interaction between a diffusible signal molecule and a transcriptional activator protein. A commonly used signal molecule is N-acylhomoserine lactones (AHLs) which differ in their sidechains (N-acyl). <em>Chromobacterium violaceum</em> is a gram-negative bacterium that can produce the characteristically purple pigment, violacein. The modified strain CV026, is inducible by synthetic AHL N-acyl side chains from C4-C8 in length, with varying degrees of sensitivity <a href="#citation7">McClean et al., 1997</a>.</p><p><em>C. Violaceum</em> was going to be utilized for its capability to show the presence of a large variety of AHLs. This would be a quick indicator for any production of AHLs without having to test protein expression in Western blot. The performed experiments with C. Violaceum tested the needed levels of synthetic AHL for our cultures.</p><p><a href="https://drive.google.com/file/d/17IBCJ6UvMP6rLA7Zu7AN4d03Ve2kcd9Q/view?usp=sharing">Chromobacterium protocol (GOOGLE DRIVE)</a></p><h1>MFC</h1><h2>Introduction</h2><p>A Microbial Fuel Cell (MFC) works similarly to a hydrogen fuel cell, but with microbes as the terminal electron donor. The decision to use MFCs as a biosensor in our project, and the people who influenced our design are outlined in our <a href="https://2020.igem.org/Team:Stockholm/Human_Practices">Human Practices</a> page.</p><h2>Hydrogen Fuel Cells</h2><p>The hydrogen fuel cell generates electricity by turning hydrogen and oxygen into water via the following reaction, which is the reverse of the electrolysis reaction:</p><p>$$ 2H_2 + O_2 \rightarrow 2H_2O + energy $$ This reaction occurs in two redox half-reactions. At the anode: $$ H_2 \rightarrow H^+ + 2e^- $$ and the cathode half-reaction: $$ \frac{1}{2} O_2 + 2H^+ + 2e^- \rightarrow 2H_2O $$</p><p>By constructing two chambers where one has access to hydrogen and the other to oxygen, the two half-reactions can take place in different locations as long as the other components (protons and electrons) can travel between the chambers. A selective membrane can be used that only allows protons to permeate to create an abundance of electrons at the anode. A conductive wire can connect the anode and cathode electrodes, which allows the electrons to pass through. The electrons cannot pass through any other way than the wire, and it is from that which the electricity in the form of direct current is generated <a href="#citation8">Felseghi et al., 2019</a>.</p><h2>Bacteria for producing electricity</h2><p>In many biological systems, the energy used is in the form ATP which is formed by breaking down nutrients. The energy extracted from glycolysis and the Krebs cycle is not primarily in the form of ATP, but NADH, which is then used in the electron transport chain in oxidative phosphorylation. In some anaerobic bacteria such as <em>S. oneidensis</em>, an extracellular electron transport pathway still exists, but without oxygen as a terminal electron acceptor to form water, oxidized metals or other compounds can be reduced instead. This can result in electricity being generated as the reduced mediator can pass its electron on <a href="https://2020.igem.org/Team:Stockholm/Experiments/citation9">Pirbadian et el., 2014</a>. <em>S. oneidensis MR-1</em> can also form highly conductive nanowires that can carry charge far outside the immediate area of the cell and remove the need for specific mediators <a href="#citation10">Reguera et al., 2005</a>. This is in <em>S. oneidensis</em> done through a set of cytochromes that carries the electrons along the nanowire <a href="https://2020.igem.org/Team:Stockholm/Experiments/citation9">Pirbadian et el., 2014</a>.</p><h2>Microbial Fuel Cells</h2><p>Bacteria such as <em>S. oneidensis</em> catalyses the reduction of surrounding metals and uses them as terminal electron acceptors to get rid of electrons in their cytochromes which allows them to complete their extracellular electron transport for energy. This is a reaction catalyzed by the redox proteins in their respiratory chain, which also pumps protons out of the cell.</p><p>In our MFC, Shenwanella Oneidensis reduces the anode directly or via nanowires and the electrons then flow through the wire, creating a current <a href="#citation11">Brutinel &amp; Gralnick</a>.</p><p>A simple design is the H-type MFC, which is structurally and conceptually similar to the hydrogen fuel cell. The hydrogen-rich anode chamber is replaced with the microbial growth chamber, where protons and electrons are generated through the extracellular electron transport rather than the metal catalysed hydrogen gas splitting. Water is formed at the air cathode where the electrons are rejoined with the protons and atmospheric oxygen.</p><div class="image"><img alt="Schematic of the H-type MFC" src="https://2020.igem.org/wiki/images/9/9a/T--Stockholm--img--Experiments--mfc.png" style="width: 100%"/><p>Figure 4: Schematic of the H-type MFC</p></div><h2>Setup</h2><p>The electricity generation of Shewanella Oneidensis was measured in a Microbial Fuel Cell (MFC). A two chamber electrochemical fuel cell was repurposed to be used for this purpose. The electrodes in each chamber consisted of stacks of 2 heat treated square graphite paper with a side length of 1.5 cm. This allowed for a tight packing of the fuel cell, while still leaving space for the media to flow between the graphite pater and allow biofilm to grow there. Separating the chambers was a Nafion proton exchange membrane. The chamber walls were made from the thickness of the rubber gasket sealing the electrodes in place. The rubber gaskets secured the membrane in place and allowed for an airtight seal around the chambers. The graphite electrodes were connected to a gold plate, where electronics could be attached. A 1 kΩ resistor was put between the electrodes and the open circuit voltage was measured to then calculate the current. A battery tester from Landt instruments was used to collect and log the data. A detailed description of the assembly can be found in the <a href="https://2020.igem.org/Team:Stockholm/Engineering#Design-MFC">Engineering</a> page.</p><div class="image"><img alt="Disassembled MFC" src="https://2020.igem.org/wiki/images/2/25/T--Stockholm--img--Experiments--MFC_Schematic.png" style="width: 100%"/><p>Figure 5: Disassembled MFC</p></div><h2>Experiments</h2><p>The MFC was inoculated with 100 uL culture of Shewanella Oneidensis MR-1 directly at the anode before assembly. The fuel cell was then fed with media from a reservoir to the anode, and air to the cathode at a constant flow at 0.1 RPM, which corresponds to approximately 21 uL/min. Two runs were performed. One long run with the wildtype strain, where the media feed was exchanged (LB and M9) to determine biofilm growth, and one run with the MtrB knockout strain without any transformed plasmid to see both biofilm growth and electricity production without the crucial MtrB gene in the MFC. Some minimal growth is however expected, as MtrE can complement the complex, although with a lower efficiency than MtrB <a href="#citation12">Coursolle &amp; Gralnick, 2012</a>. Only bacteria close to both the electrode and the proton exchange membrane should be able to generate current, so the amount of biofilm is proportional to the current produced. The media change is however done to confirm that biofilm has formed. Future experiments would include testing the electricity production of Shewanella Oneidensis MR-1 ΔMtrB with MtrB under an inducible promoter (pLux) and finally with the oscillatory circuit. The open circuit voltage was logged, which later could be converted into the current produced across the 1 kΩ resistor between electrodes.</p><div class="image"><img alt="Assembled MFC setup. Green/1: pump, Yellow/2: air reservoir, Purple/3: media reservoir, Red/4: resistor, Cyan/5: MFC" src="https://2020.igem.org/wiki/images/7/7b/T--Stockholm--img--Engineering--mfc.png" style="width: 100%"/><p>Figure 6: Assembled MFC setup. Green/1: pump, Yellow/2: air reservoir, Purple/3: media reservoir, Red/4: resistor, Cyan/5: MFC</p></div><h1>Protocols</h1><p><a href="https://2020.igem.org/wiki/images/5/57/T--Stockholm--Viability_Assay.pdf">Viability Assay</a></p><p><a href="https://2020.igem.org/wiki/images/8/8f/T--Stockholm--Ligation.pdf">Ligation</a></p><p><a href="https://2020.igem.org/wiki/images/b/ba/T--Stockholm--Restriction_digest.pdf">Restriction Digest</a></p><p><a href="https://2020.igem.org/wiki/images/b/b3/T--Stockholm--SDS_Page_and_Western_Blot.pdf">SDS Page and Western Blot</a></p><p><a href="https://2020.igem.org/wiki/images/d/da/T--Stockholm--Transformation.pdf">Heat Shock Transformation</a></p><p><a href="https://2020.igem.org/wiki/images/3/33/T--Stockholm--Colony_PCR.pdf">Colony PCR</a></p><p><a href="https://2020.igem.org/wiki/images/6/6e/T--Stockholm--Fluorescence.pdf">Fluorescence Intensity Quantification</a></p></article><article id="references"><h1>References</h1><ol><li id="citation1"><p class="author">Lodish H, Berk A, Zipursky SL, et al. (2000).</p><cite>Molecular Cell Biology. 4th edition.</cite><p><span class="details">New York: W. H. Freeman.</span></p><a class="in-text" href="#intext1">Back to text</a></li><li id="citation2"><p class="author">Wang, L., Fan, D., Chen, W., &amp; Terentjev, E. M. (2015).</p><cite>Bacterial growth, detachment and cell size control on polyethylene terephthalate surfaces.</cite><p><span class="journalTitle">Scientific Reports</span> <span class="journalInfo">5(1).</span></p><a class="in-text" href="https://doi.org/10.1038/srep15159" rel="noopener" target="_blank">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Bacterial growth, detachment and cell size control on polyethylene terephthalate surfaces." rel="noopener" target="_blank">Google Scholar</a><a class="in-text" href="#intext2">Back to text</a></li><li id="citation3"><p class="author">Horakova, K., Greifova, M., Seemannova, Z., Gondova, B., &amp; Wyatt, G. M. (2004).</p><cite>A comparison of the traditional method of counting viable cells and a quick microplate method for monitoring the growth characteristics of Listeria monocytogenes.</cite><p><span class="journalTitle">Letters in Applied Microbiology</span> <span class="journalInfo">38(3), 181-184.</span></p><a class="in-text" href="https://doi.org/10.1111/j.1472-765X.2004.01448.x" rel="noopener" target="_blank">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=A comparison of the traditional method of counting viable cells and a quick microplate method for monitoring the growth characteristics of Listeria monocytogenes." rel="noopener" target="_blank">Google Scholar</a><a class="in-text" href="#intext3">Back to text</a></li><li id="citation4"><p class="author">Welker, R. W. (2012).</p><cite>Size Analysis and Identification of Particles.</cite><p><span class="journalTitle">Developments in Surface Contamination and Cleaning</span> <span class="journalInfo">179-213.</span></p><a class="in-text" href="https://doi.org/10.1016/B978-1-4377-7883-0.00004-3" rel="noopener" target="_blank">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Size Analysis and Identification of Particles." rel="noopener" target="_blank">Google Scholar</a><a class="in-text" href="#intext4">Back to text</a></li><li id="citation5"><p class="author">Yang, P.-C., &amp; Mahmood, T. (2012).</p><cite>Western blot: Technique, theory, and trouble shooting.</cite><p><span class="journalTitle">North American Journal of Medical Sciences</span> <span class="journalInfo">4(9), 429.</span></p><a class="in-text" href="https://doi.org/10.4103/1947-2714.100998" rel="noopener" target="_blank">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Western blot: Technique, theory, and trouble shooting." rel="noopener" target="_blank">Google Scholar</a><a class="in-text" href="#intext5">Back to text</a></li><li id="citation6"><p class="author">Jarmander, J., Gustavsson, M., Do, T.-H., Samuelson, P., &amp; Larsson, G. (2012).</p><cite>A dual tag system for facilitated detection of surface expressed proteins in Escherichia coli.</cite><p><span class="journalTitle">Microbial Cell Factories</span> <span class="journalInfo">11(1), 118.</span></p><a class="in-text" href="https://doi.org/10.1186/1475-2859-11-118" rel="noopener" target="_blank">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=A dual tag system for facilitated detection of surface expressed proteins in Escherichia coli." rel="noopener" target="_blank">Google Scholar</a><a class="in-text" href="#intext6">Back to text</a></li><li id="citation7"><p class="author">McClean, K. H., Winson, M. K., Fish, L., Taylor, A., Chhabra, S. R., Camara, M., ... &amp; Williams, P. (1997).</p><cite>Quorum sensing and Chromobacterium violaceum: exploitation of violacein production and inhibition for the detection of N-acylhomoserine lactones.</cite><p><span class="journalTitle">Microbiology</span> <span class="journalInfo">143(12), 3703-3711.</span></p><a class="in-text" href="https://doi.org/10.1099/00221287-143-12-3703" rel="noopener" target="_blank">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Quorum sensing and Chromobacterium violaceum: exploitation of violacein production and inhibition for the detection of N-acylhomoserine lactones." rel="noopener" target="_blank">Google Scholar</a><a class="in-text" href="#intext7">Back to text</a></li><li id="citation8"><p class="author">Felseghi, R.-A., Carcadea, E., Raboaca, M. S., TRUFIN, C. N., &amp; Filote, C. (2019).</p><cite>Hydrogen Fuel Cell Technology for the Sustainable Future of Stationary Applications.</cite><p><span class="journalTitle">Energies</span> <span class="journalInfo">12(23), 4593.</span></p><a class="in-text" href="https://doi.org/10.3390/en12234593" rel="noopener" target="_blank">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Hydrogen Fuel Cell Technology for the Sustainable Future of Stationary Applications." rel="noopener" target="_blank">Google Scholar</a><a class="in-text" href="#intext8">Back to text</a></li><li id="citation9"><p class="author">Pirbadian, S., Barchinger, S. E., Leung, K. M., Byun, H. S., Jangir, Y., Bouhenni, R. A., ... &amp; El-Naggar, M. Y. (2014).</p><cite>Shewanella oneidensis MR-1 nanowires are outer membrane and periplasmic extensions of the extracellular electron transport components.</cite><p><span class="journalTitle">Proceedings of the National Academy of Sciences</span> <span class="journalInfo">111(35), 12883-12888.</span></p><a class="in-text" href="https://doi.org/10.1073/pnas.1410551111" rel="noopener" target="_blank">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Shewanella oneidensis MR-1 nanowires are outer membrane and periplasmic extensions of the extracellular electron transport components." rel="noopener" target="_blank">Google Scholar</a><a class="in-text" href="#intext9">Back to text</a></li><li id="citation10"><p class="author">Reguera, G., McCarthy, K. D., Mehta, T., Nicoll, J. S., Tuominen, M. T., &amp; Lovley, D. R. (2005).</p><cite>Extracellular electron transfer via microbial nanowires.</cite><p><span class="journalTitle">Nature</span> <span class="journalInfo">435(7045), 1098-1101.</span></p><a class="in-text" href="https://doi.org/10.1038/nature03661" rel="noopener" target="_blank">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Extracellular electron transfer via microbial nanowires." rel="noopener" target="_blank">Google Scholar</a><a class="in-text" href="#intext10">Back to text</a></li><li id="citation11"><p class="author">Brutinel, E. D., &amp; Gralnick, J. A. (2011).</p><cite>Shuttling happens: soluble flavin mediators of extracellular electron transfer in Shewanella.</cite><p><span class="journalTitle">Applied Microbiology and Biotechnology</span> <span class="journalInfo">93(1), 41-48.</span></p><a class="in-text" href="https://doi.org/10.1007/s00253-011-3653-0" rel="noopener" target="_blank">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Shuttling happens: soluble flavin mediators of extracellular electron transfer in Shewanella." rel="noopener" target="_blank">Google Scholar</a><a class="in-text" href="#intext11">Back to text</a></li><li id="citation12"><p class="author">Coursolle, D., &amp; Gralnick, J. A. (2012).</p><cite>Reconstruction of Extracellular Respiratory Pathways for Iron(III) Reduction in Shewanella Oneidensis Strain MR-1.</cite><p><span class="journalTitle">Frontiers in Microbiology</span> <span class="journalInfo">3.</span></p><a class="in-text" href="https://doi.org/10.3389/fmicb.2012.00056" rel="noopener" target="_blank">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Reconstruction of Extracellular Respiratory Pathways for Iron(III) Reduction in Shewanella Oneidensis Strain MR-1." rel="noopener" target="_blank">Google Scholar</a><a class="in-text" href="#intext12">Back to text</a></li></ol></article></div></div></div></main><footer><div class="container"><div class="social_media"><p>Follow iGEM Stockholm on social media</p><a href="https://www.instagram.com/igemstockholm/"><i class="fab fa-instagram"></i></a><a href="https://www.facebook.com/igemstockholm"><i class="fab fa-facebook-f"></i></a><a href="https://www.youtube.com/channel/UCh_a6JvWdh6N_i5tYFFcpyw"><i class="fab fa-youtube"></i></a><a href="https://twitter.com/igemstockholm"><i class="fab fa-twitter"></i></a><a href="https://www.linkedin.com/company/igemstockholm"><i class="fab fa-linkedin-in"></i></a><a href="https://igem.se/"><i class="fas fa-globe-europe"></i></a><a href="mailto:info@igem.se"><i class="far fa-envelope"></i></a></div><hr/><div class="sponsors"><img class="kth" src="https://2020.igem.org/wiki/images/9/94/T--Stockholm--img--Sponsors--kth.png"/><img class="ki_innovations" src="https://2020.igem.org/wiki/images/f/f1/T--Stockholm--img--Sponsors--ki_innovations.png"/><img class="mf" src="https://2020.igem.org/wiki/images/a/a0/T--Stockholm--img--Sponsors--mf.png"/><img class="snapgene" src="https://2020.igem.org/wiki/images/f/f2/T--Stockholm--img--Sponsors--snapgene.png"/><img class="neb" src="https://2020.igem.org/wiki/images/9/9c/T--Stockholm--img--Sponsors--neb.png"/><img class="idt" src="https://2020.igem.org/wiki/images/4/45/T--Stockholm--img--Sponsors--idt.png"/><img class="redbull" src="https://2020.igem.org/wiki/images/0/03/T--Stockholm--img--Sponsors--redbull.png"/><img class="benchling" src="https://2020.igem.org/wiki/images/9/98/T--Stockholm--img--Sponsors--benchling.png"/></div><hr/><div class="credits"><p>Sample template built using the iGEM Wiki Starter Pack by BITS Goa.</p><p>Code released under the MIT license.</p><p>Based on <a href="https://getbootstrap.com">Bootstrap</a> and themes <a href="https://bootswatch.com/flatly/">Flatly</a> and <a href="https://bootswatch.com/darkly/">Darkly</a> from <a href="https://bootswatch.com/">Bootswatch</a>.</p><p>Some content from the <a href="https://2020.igem.org/Team:Example">iGEM Example Wiki</a>. Images from <a href="https://unsplash.com">Unsplash</a>. Web fonts from <a href="https://fonts.google.com">Google</a>.</p></div></div></footer></div><script src="https://2020.igem.org/Template:Stockholm/content-bundleJS?action=raw&amp;ctype=text/javascript"></script></body></html>