<!-- # TODO: #6 Fix table caption font--><!-- # TODO: #7 Fix citations links font size--><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Results | iGEM Stockholm</title><script src="https://2020.igem.org/common/MathJax-2.5-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><link href="../css/content.css" rel="stylesheet"></head><body><!-- # TODO: #6 Fix table caption font--><!-- # TODO: #7 Fix citations links font size--><nav class="navbar navbar-expand-xl fixed-top"><div class="container d-flex justify-content-between"><a class="navbar-brand d-lg-inline-block" href="//"></a><button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"><i class="fas fa-bars"></i></button><div class="collapse navbar-collapse" id="navbarNav"><ul class="navbar-nav ml-auto"><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarProjectDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Project</a><div class="dropdown-menu fade-up" aria-labelledby="navbarProjectDropdown"><a class="dropdown-item" href="//Description">Description</a><a class="dropdown-item" href="//Design">Design</a><a class="dropdown-item" href="//Proof_Of_Concept">Proof of Concept</a><a class="dropdown-item" href="//Implementation">Implementation</a><a class="dropdown-item" href="//Contribution">Contribution</a><a class="dropdown-item" href="//Poster">Poster</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarLab &amp; PartsDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Lab &amp; Parts</a><div class="dropdown-menu fade-up" aria-labelledby="navbarLab &amp; PartsDropdown"><a class="dropdown-item" href="//Model">Model</a><a class="dropdown-item" href="//Engineering">Engineering</a><a class="dropdown-item" href="//Parts">Parts</a><a class="dropdown-item" href="//Experiments">Experiments</a><a class="dropdown-item" href="//Results">Results</a><a class="dropdown-item" href="//Safety">Safety</a><a class="dropdown-item" href="//Notebook">Notebook</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarHuman PracticesDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Human Practices</a><div class="dropdown-menu fade-up" aria-labelledby="navbarHuman PracticesDropdown"><a class="dropdown-item" href="//Human_Practices">Human Practices</a><a class="dropdown-item" href="//Education">Education</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarAboutDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">About</a><div class="dropdown-menu fade-up" aria-labelledby="navbarAboutDropdown"><a class="dropdown-item" href="//Team">Team</a><a class="dropdown-item" href="//Collaborations">Collaborations</a><a class="dropdown-item" href="//Sponsors">Sponsors</a><a class="dropdown-item" href="//Attributions">Attributions</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarAwardsDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Awards</a><div class="dropdown-menu fade-up" aria-labelledby="navbarAwardsDropdown"><a class="dropdown-item" href="//Entrepreneurship">Entrepreneurship</a><a class="dropdown-item" href="//Sustainable">Sustainable</a><a class="dropdown-item" href="//Medal_Criteria">Medal Criteria</a></div></li></ul></div><div class="d-flex" id="themeSwitchWrapper"><i class="far fa-sun"></i><div id="themeSwitch"><label class="switch" for="themeSwitchInput"><input type="checkbox" id="themeSwitchInput"><span class="slider round"></span></label></div><i class="far fa-moon"></i></div></div></nav><div class="parent"><header class="d-flex justify-content-center align-items-center"><div class="container"><h1>Results</h1><p class="lead pl-1">A sample page for the theme.</p><a class="arrow" href="#wave"><i class="fas fa-chevron-down"></i></a><img class="topimg" src="../assets/img/Contents--topimg_results.png"><div class="wave wave1" id="wave"></div><div class="wave wave2"></div><div class="wave wave3"></div></div></header><main><div class="container"><div class="row"><div class="sidebar col-lg-3"><div class="nav" id="contents"><h5>Contents</h5><ul></ul></div></div><div class="content col-lg-9"><article><h1>MFC</h1><p>Both the wildtype strain of <em>S. oneidensis MR-1</em> (WT) and the MtrB knockout (KO) strain were cultivated in the MFC. The media was fed to the MFC from a reservoir, and the spent media from the outflow was returned to the reservoir. This was done to allow the experiment to run without much supervision, as the flow rate was still relatively high and would otherwise require refilling every day. The cathode reservoir was effectively open to the atmosphere.</p><h2>Wildtype strain</h2><p>The biofilm growth of the wildtype strain was monitored for a total of 18 days (450 hours). The cultivation can be separated into four phases: two using LB to grow and confirm the biofilm growth, and two using M9 with lactate as the media feed to investigate the growth and biofilm integrity under lactate as the only carbon source. The calculated current for the entire run is shown in figure 14.</p><div class="image"><img src="../assets/img/Results--MFC_WT2.png" alt="Measurment of current over time in red, with the different phases in cyan." style="width: 100%"><p>Figure 1: Measurment of current over time in red, with the different phases in cyan.</p></div><p>The first phase with LB as the carbon source lasted for 260 hours. The anolyte reservoir consists of 40 mL LB media. At the initial addition of the culture to the MFC anode chamber, the current is high but rapidly drops to 0 as the excess bacteria is flushed out of the cell. After several days, the current generated starts to rise above noise levels. The current increases steadily, and is at the end at 20 μA, indicating that a successful biofilm had formed in the MFC. There is however a chance that the electricity is from cells in suspension rather than a biofilm, as the media is recycled in the reservoir.</p><p>To confirm that the current generated is from a biofilm, the reservoir with 40 mL LB media is exchanged for a new 1000 mL bottle of LB. When the bottles are changed, there is a steep drop in current. It does not drop to 0 completely, but stays at around 8-10 μA for 5 hours. This could be due to cells in suspension that were contributing to the current were removed with the fresh reservoir, or it could be due to parts of the biofilm detaching when the flow is stopped during the media change, as predicted by literature. After the initial lag phase, the current generated rapidly increased over the course of 25 hours to a peak current of 130 μA. It is after that fairly constant, perhaps having entered a stationary phase due to nutrient limitation (which can occur with a reservoir), but more likely due to other limitations related to the biofilm, such as density or biofouling of the electrode.</p><p>The third phase of the MFC run with wildtype <em>Shewanella Oneidensis</em> aimed to determine whether it could survive and produce electricity in M9 minimal media with lactic acid as the only carbon source. The LB media was exchanged for 1000 mL M9 media with lactate (instead of glucose). Immediately after the media change, the current dropped rapidly, but started to plane out after a few hours.</p><p>The current generated did not recover after two days, so more lactic acid (1 mL, 10 mM) was added to the reservoir. In the following hours, the generated current increased to a steady level, indicating that substrate limitation was the cause for the low current after media change.</p><h2>Knockout</h2><p>The biofilm growth (or rather, the lack of it) of the MtrB knockout (KO) strain was monitored for 130 hours. The setup was identical to the first phase of the wildtype strain run, with the exception of more bacteria being added at inoculation.</p><div class="image"><img src="../assets/img/Results--MFC_KO.png" alt="Measurment of current over time in cyan." style="width: 100%"><p>Figure 2: Measurment of current over time in cyan.</p></div><p>The current starts off high but rapidly drops, similar to the wildtype run. Compared to the wildtype run however, the current doesn’t increase, but remains at a low level. A comparison between KO run and the initial part of the WT run can be seen in figure 16.</p><div class="image"><img src="../assets/img/Results--MFC_Comparison.png" alt="Comparison between the initial part of the WT current (in red) and the KO current (in blue) over time." style="width: 100%"><p>Figure 3: Comparison between the initial part of the WT current (in red) and the KO current (in blue) over time.</p></div><h1>Cloning</h1><p>Aim of the cloning experiment was to get our choice of insert/construct into the plasmid pSB1C3. If the recombinant DNA was inserted successfully they were selected using antibiotic plates. For controls we had:</p><ol><li>A negative control plate: Just the <em>TOP10 E.coli</em> cells were plated; nothing grew hence showing our plates with the antibiotics were selective</li><li>The backbone plasmid psB1C3 with RFP in it was used to check successful ligation steps. Biobrick BBa_J04450 in psB1C3 was used as our backbone, partial restriction digest could be easily detected with this backbone and the colonies picked for colony PCR could be narrowed down.</li></ol><p>Restriction digest and ligation protocol was optimised for our backbone and gene blocks and they can be found in the <a href="https://2020.igem.org/Team:Stockholm/Notebook">notebook</a>.</p><div class="image"><img src="../assets/img/Results--cloning_1.png" alt="Transformed E. coli that took up the recombinant DNA" style="width: 100%"><p>Figure 4: Transformed E. coli that took up the recombinant DNA</p></div><p>The colonies that grew on the plate could either have the construct or could be an empty backbone. To find the correct colony a colony PCR was run. The ones with the correct size constructs run on an agarose gel are compiled here, for more information on the protocol check the <a href="https://2020.igem.org/Team:Stockholm/Experiments">experiments</a> page.</p><div class="image"><img src="../assets/img/Results--cloning_2.png" alt="Constructs with the right size for B, L, M, C, D and F" style="width: 100%"><p>Figure 5: Constructs with the right size for B, L, M, C, D and F</p></div><div class="image"><img src="../assets/img/Results--cloning_3.png" alt="Constructs with the right size for G, J, H and F" style="width: 100%"><p>Figure 6: Constructs with the right size for G, J, H and F</p></div><div class="image"><img src="../assets/img/Results--cloning_4.png" alt="Constructs with the right size for N,Q,P,O and plasmid 3.2" style="width: 100%"><p>Figure 7: Constructs with the right size for N,Q,P,O and plasmid 3.2</p></div><div class="image"><img src="../assets/img/Results--cloning_5.png" alt="Constructs with the right size for Q, E, H" style="width: 100%"><p>Figure 8: Constructs with the right size for Q, E, H</p></div><p>For each construct multiple right size band samples were found, these were sent out for sequencing to find the sequence with no mis alignments or alignments that gave the same amino acid sequence as an output.</p><div class="image"><img src="../assets/img/Results--cloning_6.png" alt="Sequence comparison for construct O and F. " style="width: 100%"><p>Figure 9: Sequence comparison for construct O and F.</p></div><p>12 constructs were found to have the right alignments.</p><table><caption id="table1summaryofthecloningexperiment">Table 1: Summary of the cloning experiment.</caption><thead><tr><th>Constructs</th><th>Code</th><th>Size</th><th>selection of sequence</th></tr></thead><tbody><tr><td>Constitutive promoter-RFP</td><td>Ctrl( Control)</td><td>1069</td><td>/</td></tr><tr><td>Constitutive promoter- LuxI- myc tag</td><td>A</td><td>1158</td><td>A3 but resequence end of LuxI w/ new primers</td></tr><tr><td>Promoter prmA- mcherry</td><td>B</td><td>1492</td><td>B7, B8</td></tr><tr><td>Promoter bphR1- mcherry</td><td>C</td><td>1361</td><td>C3 but mismatch in promoter and gap after it</td></tr><tr><td>Promoter Lux-GFP</td><td>D</td><td>1113</td><td>D1</td></tr><tr><td>Promoter Rhl-GFP</td><td>E</td><td>1111</td><td>E5</td></tr><tr><td>Promoter prmA-LuxI-myc tag</td><td>F</td><td>1159</td><td>F5 (F7 has a mismatch)</td></tr><tr><td>Constitutive promoter- RhlI- myc tag</td><td>G</td><td>1045</td><td>G7</td></tr><tr><td>Constitutive promoter-AiiA - myc tag</td><td>H</td><td>1189</td><td>/</td></tr><tr><td>Constitutive promoter-RhlR - myc tag</td><td>J</td><td>1299</td><td>(J3 has a mismatch)</td></tr><tr><td>RhlI-RBS-Aiia</td><td>K</td><td>1779</td><td>K5,K11</td></tr><tr><td>Constitutive promoter-bphR2 - myc tag</td><td>L</td><td>1443</td><td>L3</td></tr><tr><td>Promoter bphR1-LuxI-myc tag</td><td>M</td><td>1288</td><td>M1</td></tr><tr><td>Constitutive promoter- GFP</td><td>N</td><td>1266</td><td>N3</td></tr><tr><td>Constitutive promoter- RhlR- myc tag- Promoter Rhl-GFP</td><td>O</td><td>2107</td><td>O14,O19</td></tr><tr><td>Constitutive promoter-LuxR - myc tag</td><td>P</td><td>1328</td><td>/</td></tr><tr><td>Constitutive promoter- LuxR- myc tag- Promoter Lux-GFP</td><td>Q</td><td>2135</td><td>Q3</td></tr><tr><td>Constitutive promoter-bphR2-Myc tag -double terminator-Pbphr1--LuxI</td><td>plasmid 3.2</td><td>2493</td><td>P3.2-4 but not sure (need to resequence the middle)</td></tr></tbody></table><h1>Viability assays</h1><table><caption id="table2thefollowingpollutantsensingconstructswereusedfortheviabilityassay">Table 2: The following pollutant sensing constructs were used for the viability assay</caption><thead><tr><th>Constructs</th><th>Abbreviation</th><th>Construct Detects</th></tr></thead><tbody><tr><td>Top 10 E. coli BL21 (no plasmid)</td><td>Ctrl (Control)</td><td>N/A</td></tr><tr><td>Promoter prmA-LuxI-myc tag</td><td>F</td><td>PFOS</td></tr><tr><td>Constitutive promoter-bphR2 - myc tag - promoter bphR1-LuxI-myc tag</td><td>Plasmid 3.2 (P 3.2)</td><td>PCB/1,1-biphenyl</td></tr></tbody></table><div class="image"><img src="../assets/img/Results--fluorescence.png" alt="The growth curves of Top 10 E. coli BL21, construct F and plasmid 3.2. Samples were incubated in different concentrations of PFOS and 1,1-biphenyl for 18 h at 37°C. The red line represents the OD600 value and the blue line is after adjustment to the blank (well H7)" style="width: 100%"><p>Figure 10: The growth curves of Top 10 E. coli BL21, construct F and plasmid 3.2. Samples were incubated in different concentrations of PFOS and 1,1-biphenyl for 18 h at 37°C. The red line represents the OD600 value and the blue line is after adjustment to the blank (well H7)</p></div><div class="image"><img src="../assets/img/Results--pollutantconc.png" alt="Key to layout for figure x denoting samples and pollutant concentrations." style="width: 100%"><p>Figure 11: Key to layout for figure x denoting samples and pollutant concentrations.</p></div><p>The viability assay shows that our host cells with or without the pollutant sensing constructs are not severely affected by the addition of pollutants in terms of their population growth. An effect on the growth curve can be seen for the highest concentrations for 1,1-biphenyl at 10<sup>-3</sup> M and for PFOS at 125 μM. This suggests that these concentrations may be in the range of toxicity threshold values for our bacterial cells. However, these concentrations are well above current concentrations of these pollutants in nature around the Baltic sea.</p><h1>Characterization</h1><p>To make sure that our final sequence works perfectly we methodically divided the construct and added extra sequences that would help us with characterization such as fluorescence markers or myc tags. This was useful in troubleshooting the subparts of the final construct that needs more modeling and analysis to give out the desired proteins. Different techniques were applied to characterize the various constructs outlined in <a href="https://2020.igem.org/Team:Stockholm/Experiments">Experiments</a>.</p><h1>Fluorescence</h1><p>Fluorescence intensity of the constructs was measured both in uninduced conditions; to obtain a threshold signalling level of the promoters, and in induced conditions; to assess the degree of activation of the promoter by the inducers (i.e. AHL and/or pollutants). Inducers were added to cultures along with the constructs to be tested, and incubated overnight (Table 3). Constructs not expressing fluorescent proteins were included as controls. The results are summarised in Figure x9 and 10.</p><table><caption id="table3summaryoffluorescenceintensityquantificationexperiment">Table 3: Summary of fluorescence intensity quantification experiment</caption><thead><tr><th>Construct</th><th>Abbreviation</th><th>Fluorescent protein</th><th>Induced with AHL synthase constructs</th><th>Induced with synthetic AHL</th></tr></thead><tbody><tr><td>constitutive promoter- GFP</td><td>N</td><td>GFP</td><td>N/A</td><td></td></tr><tr><td>constitutive promoter- RhlR- myc tag- promoter Rhl-GFP</td><td>O</td><td>GFP</td><td>co-culture with G</td><td>2 μM C4-HSL</td></tr><tr><td>constitutive promoter- LuxR- myc tag- promoter Lux-GFP</td><td>Q</td><td>GFP</td><td>co-culture with A</td><td>2 μM 3OC6-HSL</td></tr><tr><td></td></tr><tr><td>Constructs</td><td>Abbreviation</td><td>Fluorescent protein</td><td>Induced with pollutants</td><td></td></tr><tr><td>promoter prmA- mCherry</td><td>B</td><td>mCherry</td><td>62.5 μM PFOS</td><td></td></tr><tr><td>promoter bphR1- mCherry</td><td>C</td><td>mCherry</td><td>10-4 M 1,1-biphenyl</td><td></td></tr><tr><td>Controls</td></tr><tr><td>constitutive promoter- LuxI- myc tag</td><td>A</td><td>N/A</td><td>N/A</td><td></td></tr><tr><td>constitutive promoter- RhlI- myc tag</td><td>G</td><td>N/A</td><td>N/A</td><td></td></tr></tbody></table><div class="image"><img src="../assets/img/Results--mCherry bar graph.png" alt="Net fluorescence intensity of constructs expressing (a) mCherry and (b) GFP. The values shown represent averages of 3 technical replicates, and have been adjusted for cell density. Error bars represent standard deviation" style="width: 100%"><p>Figure 12: Net fluorescence intensity of constructs expressing (a) mCherry and (b) GFP. The values shown represent averages of 3 technical replicates, and have been adjusted for cell density. Error bars represent standard deviation</p></div><div class="image"><img src="../assets/img/Results--O14_GFP.png" alt="GFP visualisation of construct O14 under UV light." style="width: 70%"><p>Figure 13: GFP visualisation of construct O14 under UV light.</p></div><div class="image"><img src="../assets/img/Results--O19_GFP.png" alt="GFP visualisation of construct O19 under UV light." style="width: 70%"><p>Figure 14: GFP visualisation of construct O19 under UV light.</p></div><div class="image"><img src="../assets/img/Results--Q_GFP.png" alt="GFP visualisation of construct Q under UV light." style="width: 70%"><p>Figure 15: GFP visualisation of construct Q under UV light.</p></div><p>The green fluorescence intensity of constructs O and Q increased upon induction with synthetic C4-HSL and 3OC6-HSL, respectively (Figure 9a). This suggests that the synthetic AHL molecules could bind to RhlR (constitutively expressed by construct O) and LuxR (constitutively expressed by construct Q), which could subsequently induce their respective promoters (pRhl and pLux) and therefore induce higher GFP expression. Co-culturing constructs O and Q with constructs G (which constitutively expresses RhlI) and A (which constitutively expresses LuxI) respectively induced a similar increase in GFP expression. These results suggest that AHL molecules were produced by RhlI and LuxI, which could in turn bind to RhlR and LuxR and induce the specific promoters and expression of GFP. The results of this functionality assay are supported by the Western blot results, which showed the expression of Myc-tagged LuxI (A), RhlI (G) and LuxR (Q).</p><p>The expression of mCherry by constructs B and C did not increase when induced with pollutants (Figure 9b). Construct C expresses mCherry under a bphR1 promoter, which is induced by the protein bphR2 binding to biphenyl. As no bphR2 was added, the expression of bphR1 promoter could not be induced by addition of biphenyl alone. However, the expression of mCherry by construct B (under prmA promoter) was not increased upon addition of the given concentration of PFOS. The concentration of pollutants used for characterisation were selected based on the results of the viability assays, such that they do not affect the growth of bacteria significantly. For future experiments, higher concentrations of pollutants could be added to induce the constructs.</p><h1>Western Blot</h1><p>The western blot (WB) protocol was adjusted throughout the experimental period for several factors in order to optimize the protocol and ensure optimal protein expression. A positive band corresponding to the right size suggests that the gene inserted for the desired protein has been successfully translated, although functional assays are required to confirm successful protein function. This was carried out for some of the constructs through fluorescence experiments such as construct Q (LuxR) induced by the protein expressed by construct A (LuxI), see Figure 12.</p><p>Several parameters were adjusted (see <a href="https://2020.igem.org/Team:Stockholm/Engineering#Identifyaproblemtosolveati+1=troubleshooting">engineering</a> troubleshooting guide for WB) throughout the experimental period such as:</p><ul><li>Cell culture density at lysis (in order to optimize protein expression according to cell growth phase)</li><li>Concentration of inducer (pollutant or QS signalling molecule)</li><li>Increased blocking time</li><li>Increased incubation time with antibodies</li></ul><p>The results presented here are as obtained after several repeated trials. The same positive bands for the same samples appeared over several consecutive western blots and were hence reproducible (see <a href="https://2020.igem.org/Team:Stockholm/Notebook">notebook</a> for all western blot results). In general, pollutant-inducible constructs were induced with the pollutants overnight.</p><table><caption id="table4theconstructsusedforthewesternblotexperimentinducedandnotinducedbypollutantsovernight">Table 4: The constructs used for the western blot experiment, induced and not induced by pollutants overnight.</caption><thead><tr><th>Constructs</th><th>Code</th><th>Protein Tag</th><th>Induced with pollutants (o/n)</th><th>Protein Size + Myc-tag (kDa)</th><th style="text-align:center">Correct Sequence</th></tr></thead><tbody><tr><td>Constitutive promoter- LuxI- myc tag</td><td>A</td><td>c-Myc-tag</td><td>N/A</td><td>23.1</td><td style="text-align:center">Yes</td></tr><tr><td>Constitutive promoter- RhlI- myc tag</td><td>G</td><td>c-Myc-tag</td><td>N/A</td><td>23.8</td><td style="text-align:center">Yes</td></tr><tr><td>Constitutive promoter-LuxR - myc tag</td><td>P</td><td>c-Myc-tag</td><td>N/A</td><td>29.8</td><td style="text-align:center">No</td></tr><tr><td>Constitutive promoter-RhlR - myc tag</td><td>J</td><td>c-Myc-tag</td><td>N/A</td><td>29.4</td><td style="text-align:center">Partially</td></tr><tr><td>Constitutive promoter- LuxR- myc tag- Promoter Lux-GFP</td><td>Q</td><td>c-Myc-tag</td><td>N/A</td><td>29.8</td><td style="text-align:center">Yes</td></tr><tr><td>Constitutive promoter- RhlR- myc tag- Promoter Rhl-GFP</td><td>O</td><td>c-Myc-tag</td><td>N/A</td><td>28.8</td><td style="text-align:center">Yes</td></tr><tr><td>Constitutive promoter-bphR2 - myc tag</td><td>L</td><td>c-Myc-tag</td><td>N/A</td><td>35.1</td><td style="text-align:center">Yes</td></tr><tr><td>Promoter bphR1-LuxI-myc tag</td><td>M</td><td>c-Myc-tag</td><td>10-4 M 1,1-biphenyl</td><td>23.1</td><td style="text-align:center">Yes</td></tr><tr><td>Promoter prmA-LuxI-myc tag</td><td>F</td><td>c-Myc-tag</td><td>62.5 μM PFOS</td><td>23.1</td><td style="text-align:center">Yes</td></tr><tr><td>Constitutive promoter-bphR2 - myc tag - promoter bphR1-LuxI</td><td>Plasmid 3.2 (P 3.2)</td><td>c-Myc-tag</td><td>10-4 M 1,1-biphenyl</td><td>35.1</td><td style="text-align:center">Partially</td></tr><tr><td>FtsH1Myc*</td><td>Pos Control</td><td>c-Myc-tag</td><td>N/A</td><td>71.2</td><td style="text-align:center">N/A</td></tr><tr><td>Constitutive promoter- GFP</td><td>Neg Control</td><td>None</td><td>N/A</td><td>N/A</td><td style="text-align:center">Yes</td></tr></tbody></table><p>//WESTEN BLOT IMAGE HERE //CAPTION: Western Blot of samples in table 4 with positive bands for positive control, LuxI, RhlI and LuxR in that order. M1 +, F5 + and P3.2 + were induced by pollutants according to table 4.</p><p>We used a blue safe protein stain to further troubleshoot the lack of positive bands for the constructs with correct sequences. Information of whether the general protein expression in the cell was inhibited due to the gene inserts themselves being unfavourable to our host cells could be explored through general staining of the proteome. It seemed that there was no prominent difference in the proteomic expression among different constructs.</p><div class="image"><img src="../assets/img/Results--WB-Coomassie-Stain.png" alt="Blue safe protein staining of the constructs in table 4." style="width: 100%"><p>Figure 16: Blue safe protein staining of the constructs in table 4.</p></div><p>The WB results can be interpreted in several ways. One interpretation is that the constructs lacking positive bands are in fact not expressing the desired proteins despite carrying the correct genetic inserts as confirmed by sanger sequencing. There are however several other assays one would like to perform before drawing that conclusion. Future experiments in order to troubleshoot the lack of protein expression could entail:</p><ul><li>RNA extraction (in order to pinpoint where in the process of protein expression there is a problem)</li><li>Increasing the pollutant concentration for induction of constructs M, F and Plasmid 3.2.</li><li>Lysing the cells at different points in their growth phase and running a WB to analyse if protein expression is altered at different time points.</li><li>Probing for a host protein on our blots may also aid in understanding differences in total protein expression and could help in quantifying the protein expression for the positive bands.</li><li>The adjacent sequence to the Myc tag in our protein sequence might affect the AB-binding ability <a href="#citation1">Schuchner et al., 2020</a>. Redesigning the construct sequence altering the placement of the Myc tag, increasing the number of tags or replacing the tag to another such as a His-tag might hypothetically prove more favourable for some of our constructs.</li></ul><h1>Chromobacterium violaceum</h1><p>We used a stock solution of 100 μM C4-HSL and 3OC6-HSL in Milli-Q to prepare agar with varying concentrations of AHL (C4-HSL and 3OC6-HSL) A 48 well plate containing Chromobacterium and agar with AHL was prepared as follows:</p><table><caption id="table5captiongoeshere">Table 5: Caption goes here.</caption><thead><tr><th>1</th><th>2</th><th>3</th><th>4</th><th>5</th><th>6</th><th>7</th><th>8</th></tr></thead><tbody><tr><td>control (no cells), C4-HSL</td><td>control, C4-HSL</td><td>0.1 μM, C4-HSL</td><td>0.2 μM, C4-HSL</td><td>1 μM, C4-HSL</td><td>2 μM, C4-HSL</td><td>10 μM, C4-HSL</td><td>20 μM, C4-HSL</td></tr><tr><td>control (no cells), C4-HSL</td><td>control, C4-HSL</td><td>0.1 μM, C4-HSL</td><td>0.2 μM, C4-HSL</td><td>1 μM, C4-HSL</td><td>2 μM, C4-HSL</td><td>10 μM, C4-HSL</td><td>20 μM, C4-HSL</td></tr><tr><td>control (no cells), C4-HSL</td><td>control, C4-HSL</td><td>0.1 μM, C4-HSL</td><td>0.2 μM, C4-HSL</td><td>1 μM, C4-HSL</td><td>2 μM, C4-HSL</td><td>10 μM, C4-HSL</td><td>20 μM, C4-HSL</td></tr><tr><td>control (no cells), 3OC6-HSL</td><td>control, 3OC6-HSL</td><td>0.1 μM, 3OC6-HSL</td><td>0.2 μM, 3OC6-HSL</td><td>1 μM, 3OC6-HSL</td><td>2 μM, 3OC6-HSL</td><td>10 μM, 3OC6-HSL</td><td>20 μM, 3OC6-HSL</td></tr><tr><td>control (no cells), 3OC6-HSL</td><td>control, 3OC6-HSL</td><td>0.1 μM, 3OC6-HSL</td><td>0.2 μM, 3OC6-HSL</td><td>1 μM, 3OC6-HSL</td><td>2 μM, 3OC6-HSL</td><td>10 μM, 3OC6-HSL</td><td>20 μM, 3OC6-HSL</td></tr><tr><td>control (no cells), 3OC6-HSL</td><td>control, 3OC6-HSL</td><td>0.1 μM, 3OC6-HSL</td><td>0.2 μM, 3OC6-HSL</td><td>1 μM, 3OC6-HSL</td><td>2 μM, 3OC6-HSL</td><td>10 μM, 3OC6-HSL</td><td>20 μM, 3OC6-HSL</td></tr></tbody></table><p>//INSERT CHROMOBAC GROWTH CURVE IMAGE //INSERT 48 WELL PLATE IMAGE</p><p>Readings from plate reader did not correspond to colour change in the agar wells. Only the highest concentrations of 3OC6-HSL appeared to cause a colour change. To troubleshoot this part we went on to do a preliminary test to check if there was any colour change seen in the presence of AHL. A plate with chromobacterium was grown onto which stock AHL solution was spread. Unfortunately there was no color change observed. This could possibly be because the AHL in the stock solution had degraded as it was kept at room temperature.</p><p>//INSERT IMAGE OF PLATES WITH GREEN BACKGROUND //CAPTION: I don’t know what this one shows :(:(</p><p>This experiment was repeated with chromobacterium plates using freshly prepared AHL (2µL of a 10mM AHL in acetonitrile) on plates, more for 3OC6 than C4. Purple colour was observed around the droplet areas. Also the immediate area around the droplet was found to be empty, presumably due to acetonitrile toxicity.</p><p>//INSERT IMAGE OF PLATES WITH ARTIFICIAL AHL ADDED //CAPTION: Colour change seen on chromobacterium plates at 10mM AHL</p><p>Once the concentration at which colour change was observed was found. The constructs were tested on the chromobacterium plates. Overnight cultures of A3 and G7 were tested on separate plates and a control of only chromobacterium plate was used.</p><p>//INSERT IMAGE WITH 3 PLATES //CAPTION: Control with only chromobacterium, A3+chromobacterium and Chromobacterium+G7 Not much growth or color change was observed on any.</p><p>A different experimental setup was planned with two plates on which chromobacterium and the AHL or bacteria were grown adjacent to each other. This consisted of the strains transformed e.coli A3, G7, control: top 10 E.coli, C4-HSL and 30C6-HSL.</p><p>//INSERT IMAGE OF 2 PLATES + DRAWING //CPATION: (a) plate setup for chromobacterium reaction comparison. (b) no color change observed on the plates.</p><p>There was no colour change observed on any plates; this could possibly be due to the less growth of the chromobacterium and the top 10 E.coli as the cultures were almost 24 hr grown and the cells might have died before plating. Hence from the chromobacterium assay it was seen that pure AHL at a concentration of 10mM gave a clear colour change visible to naked eyes. However in case of the constructs there wasn’t a huge difference. Although from th western blot the same constructs A and G gave clear bands multiple times it could be concluded that the proteins were being formed but the concentration of the proteins formed were too low to be detected by the chromobacterium plates prepared. Further literature review to better this experiment is needed.</p></article><article id="references"><h1>References</h1><ol><li id="citation1"><p class="author">Sch chner, S., Behm, C., Mudrak, I., &amp; Ogris, E. (2020).</p><cite>The Myc tag monoclonal antibody 9E10 displays highly variable epitope recognition dependent on neighboring sequence context.</cite><p><span class="journalTitle">Science Signaling</span> <span class="journalInfo">13(616), eaax9730.</span></p><a class="in-text" href="https://doi.org/10.1126/scisignal.aax9730" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=The Myc tag monoclonal antibody 9E10 displays highly variable epitope recognition dependent on neighboring sequence context." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext1">Back to text</a></li></ol></article></div></div></div></main><footer><div class="container"><div class="social_media"><p>Follow iGEM Stockholm on social media</p><a href="https://www.instagram.com/igemstockholm/"><i class="fab fa-instagram"></i></a><a href="https://www.facebook.com/igemstockholm"><i class="fab fa-facebook-f"></i></a><a href="https://www.youtube.com/channel/UCh_a6JvWdh6N_i5tYFFcpyw"><i class="fab fa-youtube"></i></a><a href="https://twitter.com/igemstockholm"><i class="fab fa-twitter"></i></a><a href="https://www.linkedin.com/company/igemstockholm"><i class="fab fa-linkedin-in"></i></a><a href="https://igem.se/"><i class="fas fa-globe-europe"></i></a><a href="mailto:info@igem.se"><i class="far fa-envelope"></i></a></div><hr><div class="sponsors"><img class="kth" src="../assets/img/Sponsors--kth.png"><img class="ki_innovations" src="../assets/img/Sponsors--ki_innovations.png"><img class="mf" src="../assets/img/Sponsors--mf.png"><img class="snapgene" src="../assets/img/Sponsors--snapgene.png"><img class="neb" src="../assets/img/Sponsors--neb.png"><img class="idt" src="../assets/img/Sponsors--idt.png"><img class="redbull" src="../assets/img/Sponsors--redbull.png"><img class="benchling" src="../assets/img/Sponsors--benchling.png"></div><hr><div class="credits"><p>Sample template built using the iGEM Wiki Starter Pack by BITS Goa.</p><p>Code released under the MIT license.</p><p>Based on <a href="https://getbootstrap.com">Bootstrap</a> and themes <a href="https://bootswatch.com/flatly/">Flatly</a> and <a href="https://bootswatch.com/darkly/">Darkly</a> from <a href="https://bootswatch.com/">Bootswatch</a>.</p><p>Some content from the <a href="https://2020.igem.org/Team:Example">iGEM Example Wiki</a>. Images from <a href="https://unsplash.com">Unsplash</a>. Web fonts from <a href="https://fonts.google.com">Google</a>.</p></div></div></footer></div><script src="../content.bundle.js"></script></body></html>